compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -100,12 +99,6 @@ static llvm::cl::opt<bool> clDisableArmSMETiling(
                    "target (i.e., when the +sme feature flag is present)"),
     llvm::cl::init(false));
 
-// Non-static options are used in other places.
-llvm::cl::opt<bool> clEnableTransformDialectJit(
-    "iree-llvmcpu-enable-transform-dialect-jit",
-    llvm::cl::desc("enable the usage of the transform dialect JIT"),
-    llvm::cl::init(false));
```

Comment:
Could add a bullet point to the pending release notes at https://github.com/iree-org/iree/issues/18432 mentioning this deprecation/removal. If some of this was user-facing (e.g. if flags like this were set in other repositories that we know of), could also include migration instructions.

Reply:
I went ahead and edited the release notes message there, let me know if that looks good.

Reply:
Thanks. Edited a bit:

> * The `--iree-codegen-*-transform-dialect-jit` flags have been off by default for a while and were removed across all backends: [012f8a6](https://github.com/iree-org/iree/commit/012f8a6b76dbad4f8d8ffc4a564ec300aa22a28d)